VMAccess/vmaccess.py

```diff
@@ -261,6 +261,7 @@ def _set_user_account_pub_key(protect_settings, hutil):
     user_pass = protect_settings.get('password')
     cert_txt = protect_settings.get('ssh_key')
     expiration = protect_settings.get('expiration')
+    remove_prior_keys = protect_settings.get('remove_prior_keys')
```

Comment:
Do we need to make any changes to public doc to make users aware of this feature?

Reply:
Such as here in the [readme]( https://msazure.visualstudio.com/One/_queries/edit/24518845/?queryId=03fb70b6-5356-4f0e-b52e-df420eb18713 )